src/Components/TopBar.js

```diff
@@ -27,11 +27,17 @@ const useStyles = makeStyles(theme => ({
 
 
 function TopBar(props) {
-    console.log(`Props are ${JSON.stringify(props)}`);
+  //console.log(`Props are ${JSON.stringify(props)}`);
 
 
+  const [anchorEl, setAnchorEl] = React.useState(null);
```

Comment:
This code creates a variable called anchorEl and a function called setAnchorEl and defaults anchorEl to null